doc/en/writing_plugins.rst

```diff
@@ -332,17 +332,17 @@ after others, i.e.  the position in the ``N``-sized list of functions:
 .. code-block:: python
 
     # Plugin 1
-    @pytest.hookimpl_spec(tryfirst=True)
+    @pytest.mark.tryfirst
```

Comment:
I think this behavior is deprecated, [this](http://pytest.org/latest/example/simple.html?highlight=tryfirst#post-process-test-reports-failures) suggests that this should be written as:

```python
@pytest.hookimpl(tryfirst=True)
def pytest_collection_modifyitems(items):
```